develop: update tidb cloud import steps (#10987)
pingcap/docs: develop/dev-guide-bookshop-schema-design.md

### Method 2: Via TiDB Cloud Import

> On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud (Please refer to the [Migrate From Amazon S3 or GCS](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs#step-3-import-data-into-tidb-cloud) guide for detailed steps).

Reviewer: ```suggestion
On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
```

> On the cluster detail page of TiDB Cloud, switch to the **Import** tab, and click the **Import Data** button to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud. (Please refer to the [Migrate From Amazon S3 or GCS](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs#step-3-import-data-into-tidb-cloud) guide for detailed steps):

Reviewer: ```suggestion
On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
```

Reviewer: please check if we can guide users to see more information in L140.

    arn:aws:iam::494090988690:role/s3-tidb-cloud-developer-access
    ```

>     ![The fill from step of import data](/media/develop/import-data-fill-from.png)

Reviewer: The UI is supposed to be changed today for a feature, which makes the screenshot outdated. Could we not provide the screenshot?

> 3. Click the **Next** button switch to the **File and filter** step to confirm the information of the file that will be imported.

Reviewer: ```suggestion
3. Click **Next** to go to the **File and filter** step to confirm the information of the files to be imported.
```

    ![The file and filter step of import data](/media/develop/import-data-file-and-filter.png)

> 4. Click the **Next** button again switch to the **Preview** step to confirm the sample data that will be imported is fine.

Reviewer: ```suggestion
4. Click **Next**  again to go to the **Preview** step to confirm the preview of the data to be imported.
```

    - 1,000,000 rows of rating records
    - 1,000,000 rows of order records

> 5. Click the **Start Import** button to start the import process and wait for TiDB Cloud to complete the importing.

Reviewer: ```suggestion
5. Click **Start Import** to start the import process and wait for TiDB Cloud to complete the importing.
```

    - 1,000,000 rows of rating records
    - 1,000,000 rows of order records

    ![The preview step of import data](/media/develop/import-data-preview.png)

5. Click the **Start Import** button to start the import process and wait for TiDB Cloud to complete the importing.

> For more information about TiDB Cloud, see [TiDB Cloud Documentation](https://docs.pingcap.com/tidbcloud).

Reviewer: ```suggestion
For more information about how to import or migrate data to TiDB Cloud, see [TiDB Cloud Migration Overview](https://docs.pingcap.com/tidbcloud/tidb-cloud-migration-overview).
```
